contrib/babelfishpg_tsql/src/hooks.c

```diff
@@ -4670,6 +4682,18 @@ static bool set_and_persist_temp_oid_buffer_start(Oid new_oid)
 	return true;
 }
 
+static bool
+pltsql_is_local_only_inval_msg(const SharedInvalidationMessage *msg)
+{
+	return (msg->id == SHAREDINVALRELCACHE_ID && msg->rc.local_only);
+}
+
+static EphemeralNamedRelation
+pltsql_get_tsql_enr_from_oid(const Oid oid)
+{
+	return get_ENR_withoid(currentQueryEnv, oid, ENR_TSQL_TEMP);
```

Comment:
nit: lets return right away if oid is invalid, no need to search for the entire array.

Reply:
I added a check for InvalidOid in get_ENR_withoid() on the engine side

Reply:
we can also invoke this hook across the engine, wherever we're calling get_ENR_withoid. doesn't need to be in this change, just a followup to clean up code